equinix/resource_network_device.go

```diff
@@ -899,9 +920,6 @@ func updateNetworkDeviceResource(primary *ne.Device, secondary *ne.Device, d *sc
 	if err := d.Set(networkDeviceSchemaNames["IsBYOL"], primary.IsBYOL); err != nil {
 		return fmt.Errorf("error reading IsBYOL: %s", err)
 	}
-	if err := d.Set(networkDeviceSchemaNames["LicenseToken"], primary.LicenseToken); err != nil {
```

Comment:
Since `LicenseToken` is `ForceNew`, it would never be used in Update requests (`PUT`/`PATCH`).

https://github.com/equinix/terraform-provider-equinix/pull/59/files#diff-92efd33eb689af1aff178688e7f2c8db713369942a9537f78bf4bcff7cf6e33aR244-R251

We can see that it is not used in UpdateRequests here: https://github.com/equinix/terraform-provider-equinix/pull/59/files#diff-92efd33eb689af1aff178688e7f2c8db713369942a9537f78bf4bcff7cf6e33aR1118-R1134

By removing `LicenseToken` here, we are asserting that the API returned value for LicenseToken on `GET` requests should be ignored. Is that the case? Is this field no longer returned after the network device is created with the token?

Reply:
The license token was never returned from the API. This was forcing device recreation because it was sent to empty.